puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-discovery-image-publish.yaml

```diff
@@ -0,0 +1,30 @@
+- job:
+    name: foreman-discovery-image-publish
+    node: 'admin && el && sshkey'
+    logrotate:
+      daysToKeep: 90
+      numToKeep: 30
+    scm:
+      - foreman-discovery-image
+    builders:
+      - foreman-discovery-image-publish
+    parameters:
+      - string:
+        name: proxy_repository
```

Comment:
I think this, for example, might need to be indented one level

Reply:
bah, I AM GOOD AT PROGRAMMING